.travis.yml

```diff
@@ -9,7 +9,9 @@ cache:
   directories:
     - $HOME/.m2
 install: ./mvnw install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
-script: ./mvnw verify -B
+script:
+  - ./mvnw spotless:apply
```

Comment:
I think you wanted to run `mvn spotless:check` instead of `apply`. Apply doesnt make much sense in CI as long we dont commit the changes spotless applied.

As this is already run with mvnw verify, the change is probably not needed.

Reply:
yep! good catch!